website/docs/user-group-role/user/user_basic_operations.md

```diff
@@ -9,11 +9,8 @@ The following topics are for the basic management of users: how to create, modif
 > If you want to automate user creation, you can do that either by [invitations](./invitations.md), [`user_write` stage](../../flow/stages/user_write), or [using the API](/developer-docs/api/browser).
 
 1. In the Admin interface of your authentik instance, select **Directory > Users** in the left side menu.
-
 2. Select the folder where you want to create a user.
-
 3. Click **Create** (for a default user).
-
 4. Fill in the required fields:
 
 -   **Username**: This value must be unique across your user folders.
```

Comment:
The word "folders" here feels like a new concept.  I don't see it used elsewhere. Can you clarify what it means?

Reply:
yeah it's from the user path (which is technically just a string like `foo/bar/baz` that is shown as a tree-view in the UI, and is only for organising users), I think everywhere else we just call it user path or don't mention it? I don't mind it being called folder

Reply:
I much prefer "directory" but this writing is from Baloc's original PR, and I didn't want to change it too much.